src/app_service/service/contacts/service.nim

```diff
@@ -786,12 +786,12 @@ QtObject:
 
   proc asyncContactInfoLoaded*(self: Service, pubkeyAndRpcResponse: string) {.slot.} =
     let rpcResponseObj = pubkeyAndRpcResponse.parseJson
-    let publicKey = $rpcResponseObj{"publicKey"}
+    let publicKey = rpcResponseObj{"publicKey"}.getStr
```

Comment:
This was tricky to find!
The old code was producing a string with `""` quotes! And therefore, the comparison in QML wasn't passing. 🤡